src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -0,0 +1,106 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h" /* Defines HAVE_RDMACM_QP_LESS */
+#endif
+
+#if HAVE_RDMACM_QP_LESS
```

Comment:
maybe add `if HAVE_RDMACM_QP_LESS` to `Makefile.am` instead?

Reply:
I would prefer to have this in single place - sources or Makefile.am. That's needed also [here](https://github.com/openucx/ucx/pull/3264/files#diff-31cf31fe4812ca44704203a62a055236R249)

Reply:
ok, it is not really important, but looks more clean to me. Also we do that already in other places (for instance HAVE_CUDA in test/gtest/Makefile.am and thru the source code)